sidecar-injector/main.go

```diff
@@ -0,0 +1,58 @@
+package main
```

Comment:
shouldn't this go in `sidecar-injector/cmd/sidecar-injector/main.go`?

Reply:
addressed